surveys/2023-annual-survey/questions.md

```diff
@@ -0,0 +1,769 @@
+- It allows us to build relatively correct and bug free software
+- We find it enjoyable or fun to program in Rust
+- We already know Rust so it's our default choice
+- We find it easy to prototype with
+- We must interact with existing Rust code
+
+> **justification**
+>
+> The Rust community and potential adopters of Rust have a lot of assumptions of why one would choose Rust for a project.
+> This question can help confirm or challenge our assumptions and see how they change over time.
+
+
+### Which of the following statements apply to your experience using Rust at work?
+
+Type: select all that apply (optional)
+
+Statements:
+
+- Using Rust has helped us achieve our goals
+- Adopting Rust has been challenging
+- Overall, adopting Rust has slowed down our team
+- Using Rust has been worth the cost of adoption
+- We're likely to use Rust again in the future
+
+> **justification**
+>
+> Future potential adopters may want to know how often other's have encountered success.
+
+### What about your usage of Rust has been challenging?
+
+Type: free form (optional)
+
+> **justification**
+>
+> This an opportunity to learn from adopters at companies what they struggle with when adopting Rust.
+
+
+### To what extent is Rust currently being used by your company?
+
+Type: select one
+
+- My company makes non-trivial use of Rust (e.g., used in production or in significant tooling)
+- My company has experimented with Rust or is considering using it
+- My company has not seriously considered Rust for any use [`NEXT`](#approximately-how-many-total-developers-does-your-company-employ)
+- I am unsure whether my company has considered using or currently uses Rust [`NEXT`](#approximately-how-many-total-developers-does-your-company-employ)
+
+> **justification**
+>
+> We want to establish how reliant companies are on Rust.
+
+
+### In what technology domain(s) is Rust used at your company?
+
+If you've previously answered that your company is not actively using Rust, you can leave this question blank.
+
+Type: select all that apply (optional)
+
+- Audio programming
+- Blockchain
+- Cloud computing applications
+- Cloud computing infrastructure or utilities
+- Command Line Interfaces (CLI)
+- Computer graphics
+- Computer games
+- Computer networking
+- Computer security
+- Data science
+- Database implementation
+- Desktop computer application frontend
+- Desktop computer or mobile phone libraries or services
+- Distributed systems
+- Embedded devices (with operating systems)
+- Embedded devices (bare metal)
+- HPC (High-performance [Super]Computing)
+- IoT (Internet of Things)
+- Machine learning or AI
+- Mobile phone application frontend
+- Programming languages and related tools (including compilers, IDEs, standard libraries, etc.)
+- Robotics
+- Scientific and/or numerical computing
+- Server-side or "backend" application
+- Simulation
+- Web application frontend
+- WebAssembly
+- Other (open response)
```

Comment:
Video games are missing.

Reply:
see [line 463](https://github.com/rust-lang/surveys/pull/234/files/810c9d933335b7ea63d6f10cdd8e319dee5a1c01#diff-122b0c69438c93ea71d977f69ca19ad9143b6d7a97860906aa903e9984d64985R463).

(maybe wording subject to change if you didn't spot that)